<launch>
    
    <arg name="PATH" value="/home/goudan/srbot_ws-C/src/sr_voice" />
    
    <node pkg="sr_voice"    type="XttsVoice_node"   name="XttsVoice_node"   output="screen">
        <!--                     通用设置                       -->
        <param name="GM_appid"   type="str"              value="6597fc2f" />            <!--appid         应用id-->
        <param name="GM0"        type="str"              value="purextts" />            <!--engine_type   引擎类型-->
        <param name="GM1"        type="str"              value="xiaoyan" />             <!--voice_name    发音人-->
        <param name="GM2"        type="str"              value="50" />                  <!--speed         语速-->
        <param name="GM3"        type="str"              value="60" />                  <!--volume        音量-->
        <param name="GM4"        type="str"              value="66" />                  <!--pitch         语调-->
        <param name="GM5"        type="str"              value="0" />                   <!--rdn           数字发音-->
        <param name="GM6"        type="str"              value="UTF8" />                <!--text_encoding 文本编码格式-->
        <param name="GM7"        type="str"              value="16000" />               <!--sample_rate   合成音频采样率-->
        <param name="GM_filename"   type="str"           value="/xtts_sample.wav" />    <!--filename      合成音频名称-->
        <param name="GM_filepath"   type="str"           value="$(arg PATH)/bin/wav" />    <!--filepath  合成音频存放路径-->
        <param name="GM_JsonPath"   type="str"           value="$(arg PATH)/param/BeginParam.json" />    <!--GM_path  json参数文件存放路径-->

        <!--                     离线设置                       -->
        <param name="LOCAL0"        type="str"           value="1" />                   <!--rcn             1的中文发音-->
        <param name="LOCAL1"        type="str"           value="1" />                   <!--speed_increase  语速增强-->
        <param name="LOCAL2"       type="str"            value="0" />                   <!--effect          合成音效-->
        <param name="LOCAL3"       type="str"            value="fo|$(arg PATH)/bin/msc/res/xtts/" />    <!--tts_res_path 合成资源路径-->

        <!--                     在线设置                       -->
        <param name="CLOUD0"       type="str"            value="0" />                  <!--background_sound 背景音-->
        <param name="CLOUD1"       type="str"            value="speex-wb;7" />         <!--aue              音频编码格式及压缩等级-->
        <param name="CLOUD2"       type="str"            value="text" />               <!--ttp              文本类型-->
    </node>
</launch>
